docs/src/configuring_zed.md

```diff
@@ -956,14 +956,70 @@ See Buffer Font Features
 
 ## Theme
 
-- Description: The name of the Zed theme to use for the UI.
+- Description: The name and mode of the Zed theme to use for the UI.
 - Setting: `theme`
+- Default:
+
+```json
+"theme": {
+  "mode": "dark",
+  "dark": "One Dark",
+  "light": "One Light"
+},
+```
+
+### Mode
+
+- Description: Specify theme mode.
+- Setting: `mode`
+- Default: `dark`
+
+**Options**
+
+1. Set the theme to dark mode
+
+```json
+{
+  "mode": "dark"
+}
+```
+
+2. Set the theme to light mode
+
+```json
+{
+  "mode": "light"
+}
+```
+
+3. Set the theme to system mode
+
+```json
+{
+  "mode": "system"
+}
+```
+
+### Dark
+
+- Description: The name of the dark Zed theme to use for the UI.
+- Setting: `dark`
 - Default: `One Dark`
 
 **Options**
 
 Run the `theme selector: toggle` action in the command palette to see a current list of valid themes names.
```

Comment:
Do we need this in every topic? This is the same as line 1021.

Reply:
@subashcs Perhaps it would be clearer to keep only one, I just wanted to clarify that this is an option for both the `Dark` and `Light` settings.